tests/unit/spec/base/mozilla-accordion.js

```diff
@@ -14,33 +15,33 @@ describe('mozilla-accordion.js', function () {
-            '<div id="accordion1" class="accordion">',
-                '<section>',
-                    '<h2 data-accordion-role="tab">Section 1</h2>',
-                    '<div data-accordion-role="tabpanel"><p>Section 1 contents.</p></div>',
-                '</section>',
-                '<section>',
-                    '<h2 data-accordion-role="tab">Section 2</h2>',
-                    '<div data-accordion-role="tabpanel"><p>Section 2 contents.</p></div>',
-                '</section>',
-                '<section>',
-                    '<h2 data-accordion-role="tab">Section 3</h2>',
-                    '<div data-accordion-role="tabpanel"><p>Section 3 contents.</p></div>',
-                '</section>',
-            '</div>',
-            '<div id="accordion2" class="accordion">',
-                '<section>',
-                    '<h2 data-accordion-role="tab">Section 1</h2>',
-                    '<div data-accordion-role="tabpanel"><p>Section 1 contents.</p></div>',
-                '</section>',
-                '<section>',
-                    '<h2 data-accordion-role="tab">Section 2</h2>',
-                    '<div data-accordion-role="tabpanel"><p>Section 2 contents.</p></div>',
-                '</section>',
-                '<section>',
-                    '<h2 data-accordion-role="tab">Section 3</h2>',
-                    '<div data-accordion-role="tabpanel"><p>Section 3 contents.</p></div>',
-                '</section>',
+            '<div id="accordion1" class="accordion">' +
+                '<section>' +
+                    '<h2 data-accordion-role="tab">Section 1</h2>' +
+                    '<div data-accordion-role="tabpanel"><p>Section 1 contents.</p></div>' +
+                '</section>' +
+                '<section>' +
+                    '<h2 data-accordion-role="tab">Section 2</h2>' +
+                    '<div data-accordion-role="tabpanel"><p>Section 2 contents.</p></div>' +
+                '</section>' +
+                '<section>' +
+                    '<h2 data-accordion-role="tab">Section 3</h2>' +
+                    '<div data-accordion-role="tabpanel"><p>Section 3 contents.</p></div>' +
+                '</section>' +
+            '</div>' +
+            '<div id="accordion2" class="accordion">' +
+                '<section>' +
+                    '<h2 data-accordion-role="tab">Section 1</h2>' +
+                    '<div data-accordion-role="tabpanel"><p>Section 1 contents.</p></div>' +
+                '</section>' +
+                '<section>' +
+                    '<h2 data-accordion-role="tab">Section 2</h2>' +
+                    '<div data-accordion-role="tabpanel"><p>Section 2 contents.</p></div>' +
+                '</section>' +
+                '<section>' +
+                    '<h2 data-accordion-role="tab">Section 3</h2>' +
+                    '<div data-accordion-role="tabpanel"><p>Section 3 contents.</p></div>' +
+                '</section>' +
```

Comment:
Wait so, maybe I am not reading this correctly but, seeing you are now creating a string by concatenation, does this still need the array and the call to `join`?